docker/include/bin/start_faux

```diff
@@ -132,9 +132,12 @@ elif [ -n "${options[bacnetfail]}" ]; then
          FauxDeviceEngine.EntryPoint $local_ip $broadcast_ip "Faux-Device-Fail.json"  &
 fi
 
-if [ -n "${options[ntp_client]}" ]; then
+if [ -n "${options[ntpv4fail]}" ]; then
```

Comment:
can you name the options something to do with how the behavior is different, not the pass/fail result?  Faux itself doesn't really "know" about what the test is, it just knows what its behavior is.